CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.6.0
+* Update all packages to be aligned with each other and compatible with the supported sdk/flutter version
```

Comment:
nit: remove this and I will add it manually when releasing a package

Reply:
i have removed it, should i decrease the plugin versions as well?

Reply:
Yes, it would be great to keep it unchanged - I have a script to handle it when releasing versions.

Reply:
+1 i have removed it